src/screens/CarouselScreen.js

```diff
@@ -37,6 +42,7 @@ class CarouselScreen extends Component {
   _renderItem({item, index}) {
     return (
       <View
+        testID={item.itemName}
```

Comment:
it makes sense to use `utils/automationIDs` to generate cross-platform IDs

Reply:
done